### PR TITLE
Add run description

### PR DESCRIPTION
**Pull request by nxvl**
@rainforestapp/devs

### CODE REVIEW

File: lib/rainforest/cli/options.rb [hunk 1]
```diff
@@ -67,10 +67,15 @@ def initialize(args)
             @custom_url = value
           end
 
+          opts.on("--run-description DESCRIPTION", "Add a description for the run been started.") do |value|
```
Comment (jipiboily) [inline lib/rainforest/cli/options.rb]: `Add a description for the run been started` -> `Add a description for the run.`

**Comment by jipiboily** [comment]
@nxvl please update the README and ask @shosti to review so a native English speaker review the README and the argument help :)

**Comment by nxvl** [comment]
Updated the REAME, @ukd1 @shosti please review my not that good english please!

**Comment by shosti** [comment]
:+1:

**Pull request merged**